libyul/optimiser/SSATransform.cpp

```diff
@@ -151,3 +210,14 @@ void SSATransform::run(Block& _ast, NameDispenser& _nameDispenser)
 	SSATransform{_nameDispenser, assignments.names()}(_ast);
 }
 
+Statement SSATransform::assignVarToSSAVar(YulString _varName, SourceLocation _loc)
```

Comment:
This can now be used in other places, too.

Reply:
Can it? non-SSA variables doesn't get assigned back to SSA variable in other places.

Reply:
At least there is some overlap with https://github.com/ethereum/solidity/pull/7379/files#diff-be777172e6369d562fbe2e09f3d51039R137